install_config/configuring_authentication.adoc

```diff
@@ -290,6 +290,7 @@ oauthConfig:
     provider:
       apiVersion: v1
       kind: KeystonePasswordIdentityProvider
+      url: https://keystone_url:5000/v2.0
```

Comment:
make this follow the format of other example urls (see other places in the doc). Also, is the v2.0 right? I thought it had to be a v3 server